### ISSUE TITLE
Windows Install Issue

### ISSUE THREAD

**Issue by Qriist** [transitive-bullshit/ffmpeg-concat#38]
Hello, thank you for the software. I know it takes time to develop quality code.

I cannot get past the basic install of ffmpeg-concat, it appears to request a file that simply doesn't exist, a `package.json`. My initial attempt was through cygwin (my default building environment), then through an elevated cmd prompt. Both appear to have the same error. 

While I'm more or less proficient in building files from source I have installed npm just for this bit of code; I am completely unfamiliar with the environment's expectations.

Below is the entire log. Thanks in advance for your time.
[2019-02-28T04_53_53_929Z-debug.log](https://github.com/transitive-bullshit/ffmpeg-concat/files/2913476/2019-02-28T04_53_53_929Z-debug.log)

**Comment by KosayJabre** [comment]
It seems this project's been dead for a while ... I have not been able to install it either

**Comment by transitive-bullshit** [comment]
I apologize that you're having issues initializing the lib, but I can verify that it is definitely still working and maintained. It is being used in production at several companies.

If you have a build issue, please provide more info about your build environment such as OS version, Node.js version, and exact reproduction steps.

@Qriist from your build log, it looks like a native `node-gyp` error meaning that some transitive dependency isn't compiling on your platform. I don't have a Windows machine, so it's difficult for me to test this.

**Comment by Qriist** [comment]
Alrighty! Here's what I did to resolve my issues. I then uninstalled everything that I had tried messing with and did it again (with just these steps) to verify. After everything was uninstalled I tried calling all of them on the console, just to ensure removal.

All steps were on Win10 x64.
1. Install Python 2.7 (I used the Chocolatey shim but the regular install should work.)
2. Install ffmpeg ( I chose to use this "all options" package: http://johnwarburton.net/blog/?p=28459 )
3. Install nvm-windows ( https://github.com/coreybutler/nvm-windows )
4. Open an Admin console window
5: `nvm install 8` ( or 9 )
6: `npm i ffmpeg-concat`
7: This command will mux your MKVs into MP4 containers for proper parsing by ffmpeg-concat:
`for /R %f IN (*.mkv) DO ffmpeg -i "%f" -c copy "%~nf.mp4"` (Obviously, adjust to your needs)

Once the files were prepared I successfully encoded the provided test files and some 1080p clips I extracted from a bluray.

@transitive-bullshit I hope that helps future users. The only other thing I am having issue with is the json file for multiple transitions. I'll open another ticket since that is unrelated to this one.

@KosayJabre pinging you so you see this.
